### PR TITLE
Fo accordion a11y #65 [staging]

### PR DESCRIPTION
**Pull request by olufemioye**
# Description
- Updated accordion style.
- Added a11y attribute to the documentation.

Fixes https://github.com/vc4a-dev/plan/issues/65

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: scss/02-molecule/_accordion.scss [hunk 1]
```diff
@@ -1,39 +1,42 @@
...
       }
     }
-
-    &:active {
-      background: var(--hover-ui);
-    }
   }
 
   &__title {
+    display: block;
```
Comment (matt-vc4a) [inline scss/02-molecule/_accordion.scss:23]: is this going to work with the flex-grow?

Maybe one of them needs to be removed

File: docs/molecules/accordion.html [hunk 2]
```diff
@@ -127,64 +127,76 @@ <h1>Accordion</h1>
...
-                            <p class="accordion__title label-lg">Accordion one</p>
+                    <div class="accordion is-active" id="accordion_1">
+                        <div class="accordion__header" aria-expanded="true" data-toggle="toggle" data-target="#accordion_1">
+                            <span class="accordion__title label-lg">Accordion one</span>
                             <svg class="icon accordion__close">
                                 <use href="#plus"></use>
                             </svg>
                         </div>
-                        <div class="accordion__body">
+                        <div class="accordion__body" aria-expanded="true">
```
Comment (matt-vc4a) [inline docs/molecules/accordion.html:137]: So this is the thing I was saying about us being wrong about where to put aria-expanded

It should be on the button you click, not the element that is expanded.

Also you need the JS to change it at the appropriate time (I don't see the JS being updated here)

image







### [ [documentation link](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded) ]

**Review by matt-vc4a — CHANGES_REQUESTED** [review]

File: scss/02-molecule/_accordion.scss (hunk 1 above)
Comment (olufemioye) [reply scss/02-molecule/_accordion.scss:23]: This will work. I added `block` as a display property because the default tag is now a `span`.

**Comment by xoref** [comment]
@olufemioye There is no PR for production yet?

File: docs/molecules/accordion.html (hunk 2 above)
Comment (xoref) [reply docs/molecules/accordion.html:137]: it's a bit confusing tho after reading this:
> For example, `aria-expanded` is applied **to the parent item** in a DOM tree to indicate whether its child branch is shown. The parent controls the visibility of the associated child branch, as well.

Comment (matt-vc4a) [reply docs/molecules/accordion.html:137]: https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions

image

**Pull request merged**